Sources/SwiftProtobuf/TextFormatScanner.swift

```diff
@@ -1243,40 +1253,40 @@ internal struct TextFormatScanner {
 
     /// Helper to see if this could be the start of a hex or octal number so unknown field
     /// value parsing can decide how to parse/validate.
-    private func isNextNumberFloatingPoint() -> Bool {
-        // NOTE: If we run out of characters can can't tell, say it isn't and let
-        // the other code error handle.
+    private func mustParseNumberAsDecimal() -> Bool {
+        // NOTE: If we run out of characters/can't tell; then just say it doesn't have
+        // to be decimal, and let the other code error handle it.
         var scan = p
         var c = scan[0]
 
-        // Floats for decimals can have leading '-'
+        // Floats or decimals can have leading '-'
         if c == asciiMinus {
             scan += 1
             if scan == end { return false }
             c = scan[0]
         }
 
         if c == asciiPeriod {
-            return true  // "(-)." : clearly a float
+            return false  // "(-)." : clearly a float
```

Comment:
So TextFormat doesn't accept `-.123` or `.123` as a floating-point number?

Reply:
Oh, I see.  This function is now checking for a decimal integer.  That's why the true/false swap.